components/contact-details.tsx

```diff
@@ -38,9 +39,16 @@ const DescriptionItem = (props: DescriptionItemProps) => {
     <div className="py-4 px-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4">
       <dt className="text-sm font-medium text-gray-500">{props.label}</dt>
       <dd className="mt-1 flex text-sm text-gray-900 sm:mt-0 sm:col-span-2">
-        <span className="flex-grow">
-          {htmr(value, { transform: htmrTransform })}
-        </span>
+        {props.isLink && value.length > 0 ? (
+          <span className="overflow-y-scroll flex-grow whitespace-nowrap copybox border-b border-brand-500 mr-2">
```

Comment:
Use `overflow-y-auto` instead of `overflow-y-scroll`. We want to hide the scrollbar when it's not needed for a better user experience.